### ISSUE TITLE
Vod chats not working anymore

### ISSUE THREAD

**Issue by paontuus** [alamminsalo/orion#224]
The VOD chat on the client doesn't work anymore on Windows and MacOS, tried this out on 3 different systems.

**Comment by ahjolinna** [comment]
yeah it has been broken for a while now also on Linux, its API change...again

**Comment by ahjolinna** [comment]
this seems to have been fixed in the "-git" version, don't know when next stable version will arrive to win/mac

**Comment by rakslice** [comment]
Dupe of #209
